form/form_for.go

```diff
@@ -127,7 +127,7 @@ func (f FormFor) addFormatTag(field string, opts tags.Options) {
 
 //RadioButton creates a radio button for a struct field
 func (f FormFor) RadioButton(field string, opts tags.Options) *tags.Tag {
-	log.Println("[Warning] RadioButton is deprecated and may be removed in the future, use RadioButtonTag instead")
+	// log.Println("[Warning] RadioButton is deprecated and may be removed in the future, use RadioButtonTag instead")
 	return f.RadioButton(field, opts)
```

Comment:
I might be wrong, but isn’t this just calling itself?

Reply:
ouch, you're completely right

Reply:
I wonder if we ever called this function from the tests 🤔, seems not, otherwise it would have had an infinite loop in here.